internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -19,21 +19,19 @@ const (
 	errorReplicationSpecIDNotSet = "replicationSpecID not set for zoneName %s"
 )
 
-type LegacySchemaInfo struct {
+type ExtraAPIInfo struct {
```

Comment:
As we need to read the LegacyInfo to populate the old `replication_spec.*.id` it didn't make sense to keep two structs. Now a single ExtraAPIInfo is used.